definitions/browser-application/golden_metrics.yml

```diff
@@ -26,3 +26,17 @@ errors:
     from: JavaScriptError
     eventId: entityGuid
     eventName: appName
+pageLoad:
```

Comment:
the other metrics have their unit as part of the metric name, e.g. `firstInputDelay75PercentileMs`, would it make sense to add that suffix to the `pageLoad` metric as well? e.g. `pageLoadMs` in case the duration is reported in milliseconds?

Reply:
Hi! I think that if I add the unit, there is no need to have the units in the title, no? So in that case, I will add the units and remove the unit as part of the metric name.